src/test/lrsql/lrs_test.clj

```diff
@@ -29,13 +29,9 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (defn- remove-props
-  "Remove properties added by `prepare-statement`."
+  "Remove properties added by `lrsql.util.statement/prepare-statement`."
   [statement]
-  (-> statement
-      (dissoc "timestamp")
-      (dissoc "stored")
-      (dissoc "authority")
-      (dissoc "version")))
+  (dissoc statement "timestamp" "stored" "authority" "version"))
```

Comment:
Not related to the dep updates but I feel like I have to address this after our previous conversations.